### ISSUE TITLE
robust to different level of noise

### ISSUE THREAD

**Issue by robbiewongBD** [zhjwustc/cvpr17_testing_matconvnet#1]
Does the network have to be trained with different level of noise？looking forward to the "train" code.

**Comment by zhjwustc** [comment]
yes, it has to be trained with different noise levels and so does most of the deep learning based methods.
